lte/gateway/docker/mme/Dockerfile

```diff
@@ -283,16 +285,16 @@ ENV C_BUILD=/build/c
 
 # Install a few tools (may not be necessary later on)
 ENV DEBIAN_FRONTEND=noninteractive
-
-# redis-server installation is temporary
-# once we resolve the issue when running it on a separate container, we will remove it
-
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get upgrade --yes && DEBIAN_FRONTEND=noninteractive apt-get install --yes \
-    psmisc \
-    openssl \
-    net-tools \
-    tshark \
-    redis-server \
+ENV TZ=Europe/Paris
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get upgrade --yes && \
```

Comment:
The `DEBIAN_FRONTEND=noninteractive` is already set above you don't need it on the next apt installs

Reply:
Will fix forward in master branch.